src/redux/greetingSlice.js

```diff
@@ -0,0 +1,45 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+const URL_API = 'http://localhost:3000/api/v1/greetings/random';
+export const fetchRandomGreeting = createAsyncThunk('greetings/fetchRandomGreeting', async () => {
+  try {
+    const response = await fetch(URL_API, {
+      headers: {
+        method: 'GET',
+        'Content-Type': 'application/json',
+      },
+    });
```

Comment:
- [ ] I noticed a small issue in your fetch request headers. You are including the method inside the headers, which is not correct. The method should be specified directly in the fetch options, not in the headers.